docs/self-managed/modeler/web-modeler/troubleshooting/troubleshoot-zeebe-connection.md

```diff
@@ -48,7 +48,7 @@ Ensure you properly [configured your Zeebe ingress to support ALPN](../../../pla
 [Inspect the connection](#how-can-i-get-details-about-a-secure-remote-connection) to understand which certificates are
 being returned by the server and ensure you configure Web Modeler for [custom SSL certificates](#how-can-i-provide-a-custom-zeebe-ssl-certificate).
 
-If intermediate signing authorities sign the server certificate, ensure the remote endpoint [serves both server and
+If intermediate signing authorities sign the server certificate, ensure the remote cluster URL [serves both server and
```

Comment:
🤔 not sure whether it makes sense to replace this, since we actually meant endpoint (like a service/deployment)

Reply:
Resolved in latest commit 👍